src/pool/rpc.h

```diff
@@ -295,8 +295,10 @@ CRT_RPC_DECLARE(pool_exclude, DAOS_ISEQ_POOL_TGT_UPDATE,
 CRT_RPC_DECLARE(pool_exclude_out, DAOS_ISEQ_POOL_TGT_UPDATE,
 		DAOS_OSEQ_POOL_TGT_UPDATE)
 
-#define DAOS_ISEQ_POOL_EVICT	/* input fields */		 \
-	((struct pool_op_in)	(pvi_op)		CRT_VAR)
+#define DAOS_ISEQ_POOL_EVICT	/* input fields */			 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
a false positive that always occurs when using macros for CaRT RPCs.